lib/glimesh/streams/channel.ex

```diff
@@ -12,13 +12,25 @@ defmodule Glimesh.Streams.Channel do
     field :language, :string
     field :thumbnail, :string
     field :stream_key, :string
+    field :inaccessible, :boolean, default: false
 
+    field :chat_rules_md, :string
+    field :chat_rules_html, :string
     timestamps()
   end
 
   def changeset(channel, attrs \\ %{}) do
     channel
-    |> cast(attrs, [:title, :category_id, :language, :thumbnail, :stream_key])
+    |> cast(attrs, [
+      :title,
+      :category_id,
+      :language,
+      :thumbnail,
+      :stream_key,
+      :chat_rules_md,
+      :inaccessible
+    ])
+    |> set_chat_rules_content_html()
```

Comment:
Can we add a validator for `chat_rules_md` to ensure it doesn't get too long?

Reply:
I'll need some help on this because currently if a changeset is invalid it breaks the entire page due to the Repo.preload in the update_channel function.

Reply:
Added back the validate clause but it will throw an internal error if the md is over 8192 characters. Will have to figure a way of showing the error generated.

Reply:
No longer errors, did some error checking internally since the function is shared between two pages.